src/archivist/recorder/index.js

```diff
@@ -1,23 +1,22 @@
-export default class Recorder {
-  constructor({ versionsStorageAdapter, snapshotsStorageAdapter }) {
-    if (!versionsStorageAdapter || !snapshotsStorageAdapter) {
-      throw new RangeError('Storage adapters should be defined both for versions and snapshots');
```

Comment:
> This validation does not seem to have been reproduced in this class nor in the Factory. It was not tested. I'm okay with losing it and considering behaviour undefined in case storage is not defined for both cases, but this is less defensive as it was.